quickwit-actors/src/progress.rs

```diff
@@ -85,7 +85,7 @@ impl Progress {
 
     pub fn protect_zone(&self) -> ProtectedZoneGuard {
         loop {
-            let previous_state: ProgressState = self.0.load(Ordering::SeqCst).into();
+            let previous_state: ProgressState = self.0.load(Ordering::Relaxed).into();
```

Comment:
Thinking about ordering is always a brainfuck. But in this simple case, I agree, I think we don't really care so `Relaxed` sounds good.